packages/nextcloud/lib/src/client.dart

```diff
@@ -26,7 +26,8 @@ class NextcloudClient extends DynamiteClient {
     final AppType appType = AppType.unknown,
     final String? userAgentOverride,
     super.cookieJar,
-  }) : super(
+  })  : assert(!baseURL.path.endsWith('/'), "The baseURL should not end with a '/'"),
```

Comment:
Should we move this check into the generated dynamite code?

Reply:
I think we should rather move it into the dynamite client. And all other clients would "inherit" the assertion.